src/SFA.DAS.ApprenticeCommitments.Jobs.Functions/ApprenticeshipCommitmentsJobsHandler.cs

```diff
@@ -19,21 +20,25 @@ public class ApprenticeshipCommitmentsJobsHandler
     {
         private readonly IEcsApi _api;
         private readonly ILogger<ApprenticeshipCommitmentsJobsHandler> _logger;
-        private readonly LoginServiceOptions _nServiceBusOptions;
+        private readonly ApplicationSettings _settings;
 
         public ApprenticeshipCommitmentsJobsHandler(
+            ApplicationSettings settings,
             IEcsApi api,
-            ILogger<ApprenticeshipCommitmentsJobsHandler> logger,
-            LoginServiceOptions nServiceBusOptions
-            )
+            ILogger<ApprenticeshipCommitmentsJobsHandler> logger
+                                                   )
         {
             _api = api;
             _logger = logger;
-            _nServiceBusOptions = nServiceBusOptions;
+            _settings = settings;
         }
 
         public async Task Handle(ApprenticeshipCreatedEvent message, IMessageHandlerContext context)
         {
+            var templateId = "4f04cf81-b291-4577-9452-ecab875ed6f8";
```

Comment:
The `das-employer-config` has been changed as part of CS-747, but this branch can't use that as it still requires the `RedirectUrl`.

This hard-coded value will be replaced by config as part of the next PR: 
https://github.com/SkillsFundingAgency/das-apprentice-commitments-jobs/blob/ae18dd93bd8e984619861894697db1f08aec80e3/src/SFA.DAS.ApprenticeCommitments.Jobs.Functions/ApprenticeshipRegisteredEventHandler.cs#L41